backend/api/pipeline.proto

```diff
@@ -214,6 +214,9 @@ message CreatePipelineVersionRequest {
   // ResourceReference inside PipelineVersion specifies the pipeline that this
   // version belongs to.
   PipelineVersion version = 1;
+
+  // Bool indicating if pipeline version should be updated by default or not
+  bool update_pipeline_default_version = 2;
```

Comment:
We are using proto3 and by default all fields are optional, but not nullable.
So there's no way to distinguish between this flag is false from this flag is not set.

You need a solution like https://stackoverflow.com/a/42634681/8745218, I'm not sure which we suggest though.
/cc @IronPan 
Can you help?